src/ducks/export/helpers.js

```diff
@@ -10,17 +10,16 @@ import { JOBS_DOCTYPE } from 'doctypes'
 export const isExportJobInProgress = async client => {
   const { data } = await client.query(
     Q(JOBS_DOCTYPE)
-      .where({
+      .partialIndex({
```

Comment:
j'avais noté qu'il n'était pas nécessaire de mettre dans `partialIndex` ET dans `indexFields`, on pourrait donc supprimer `indexFields` non ?
J'ai noté aussi de ne peut pas être seulement un `partialIndex` sans `where`...
Et enfin que tant que https://github.com/cozy/cozy-client/issues/1270 n'est pas corrigé, qu'il faut mettre au moins `_id` dans l'`indexFields` lors de l'utilisation d'un `$or` dans un `where`, mais qu'en est-il dans un `partialIndex` ? @paultranvan

Reply:
> J'ai noté aussi de ne peut pas être seulement un partialIndex sans where

on a eu une issue là dessus à un moment, mais corrigé depuis

> Et enfin que tant que https://github.com/cozy/cozy-client/issues/1270 n'est pas corrigé,

L'issue précise bien que c'est quand on ne précise pas de `.indexFields`, ce qui est le cas ici

Reply:
> Il y a besoin du indexFields pour le sort

on parle de quel sort ?

> on a eu une issue là dessus à un moment, mais corrigé depuis

top merci 👍 

> L'issue précise bien que c'est quand on ne précise pas de .indexFields, ce qui est le cas ici

Je comprends pas bien, il y a bien un `.indexFields` ici non ? 🤔 

De plus j'avais noté que `.indexFields` est à mettre systématiquement. C'est donc faux dans certain cas ?

Reply:
> Je comprends pas bien, il y a bien un .indexFields ici non ? 
De plus j'avais noté que .indexFields est à mettre systématiquement. C'est donc faux dans certain cas ?

Oui il y a un indexFields, donc l'issue ne s'applique pas ici. L'indexFields n'est pas obligatoire dans le sens où ça peut marcher sans grâce à la magie de cozy-client. Mais on encourage fortemment à le mettre, car autrement ça peut amener des bugs comme l'issue que tu as mentionné